src/reducers/jugglingTrickSlice.js

```diff
@@ -3,9 +3,7 @@ import jugglingTricks from "../assets/JugglingTricks/JugglingTricks.json";
 
 export const jugglingTricksSlice = createSlice({
   name: "jugglingTricks",
-  initialState: {
-    value: jugglingTricks,
-  },
+  initialState: jugglingTricks,
```

Comment:
My understanding is that using the value is best practice, however that could be when it's an individual value?

Reply:
It depends on the state you are trying to store, `value` is arbitrary. In our case since this reducer is to get our juggling tricks we can just setup our state w/ the tricks instead of putting it a layer down.

Reply:
so when is value not arbitrary? 

when we have reducers as well?

Reply:
It typically depends on the data you are trying to store. Usually you would keep things compartmentalized  and bring them together in the store.

In one example we could have a state that tracks user and pro data

initialState: {
    userData: [{name: 'john smith', address: 123 street}, {}. {} ],
    proData: [{businessName: 'AppleSauce', address: 123 gala ln}, {}, {}],
    holidaySales: ['July 4th', 'January 1st']
}

A better way to format this would be to have separate reducers for each, and each one of those would contain it's own state. Then in the store, we combine the reducers- then we can use `useSelector((state) => state.users)` or state.pros, or state.holidays.

Lemme know if that makes sense I know I am very rusty on my explanation skills and could use more practice with that muscle.

Reply:
I see what you're saying works for me :)